src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -902,6 +936,13 @@ bool MainWindow::can_close_project()
     return false;
 }
 
+bool MainWindow::can_close_project()
```

Comment:
I separated it to use in python module, but don't use it now. Should I revert it?